.eslintrc

```diff
@@ -0,0 +1,21 @@
+{
+  "ecmaFeatures": {
+    "jsx": true
+  },
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    "quotes": 1,
+    "no-trailing-spaces": 1,
+    "strict": 1,
```

Comment:
My understanding is that the semantics of the language become stricter when you enable strict mode (oddly enough :P). In particular, something like `var blah1 = "thing"; console.log( blah2 );` would become an error rather than a warning. 

That being said, I'm not entirely certain if this is something that's still relevant in ES6 - I know it certain is for ES5. I'll do a bit more digging and see what the verdict is.